### DISCUSSION TITLE
v0.99.9 HF4 - Quectel RM520N-GL in USB Sled

### DISCUSSION THREAD

**Discussion by netninja71** [SmoothWAN/SmoothWAN#129] [General & Help]
Good afternoon,
       I'm on the latest X86 version of SmoothWAN, and I've noticed that USB modems in MBIM or QMI mode don't seem to work like they did in previous versions.  I know there was some work done in this latest release in regards to USB devices, but I need some guidance on how to set up a USB MBIM modem now.  It looks like the changes worked great for Android tethering, it's working perfect.

**Comment by netninja71** [comment]
To add some more info, it looks like the USB Modem in both MBIM and QMI modes is created as a "DHCP Client" interface automatically, and changing it to either "Modem Manager" or "QMI Cellular" doesn't work.

**Reply by TalalMash** [reply]
Hi, sorry about that, if you're experienced enough with OpenWrt delete `/etc/hotplug.d/net/99-usbnamer` 

Example in terminal:
`rm /etc/hotplug.d/net/99-usbnamer` 

I'll use a different form of detection in next release with a new UI.

**Reply by TalalMash** [reply]
Note that if you use PBR, you need to add the interface name of the newly created profile in `Supported Interfaces` (Advanced tab).

**Comment by netninja71** [comment]
Thank you! I’ll work on this when I get home from work!



On Wed, Mar 8, 2023 at 3:29 PM Talal ***@***.***> wrote:

> Note that if you use PBR, you need to add the interface name of the newly
> created profile in Supported Interfaces (Advanced tab).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/SmoothWAN/discussions/129#discussioncomment-5246598>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4FLOXJJI34ZTQXK65OPJFLW3D25JANCNFSM6AAAAAAVUGMJEQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
--
